gateway/ln-gateway/src/actor.rs

```diff
@@ -229,6 +229,33 @@ impl GatewayActor {
             .map(|out_point| out_point.txid)
     }
 
+    pub async fn backup(&self) -> Result<()> {
+        self.client
+            .mint_client()
+            .back_up_ecash_to_federation()
+            .await
+            .map_err(LnGatewayError::Other)?;
+
+        Ok(())
+    }
+
+    pub async fn restore(&self) -> Result<()> {
```

Comment:
should be calling restore on actor start. Just in case there was previous backed up state?

Reply:
No. That's not the intended use and it might be way too slow. In normal circumstances the client should track their state on their own.

Reply:
Yea. Restore is when you lost your gateway client db and want to restore it from seed phrase. Should be done manually by the CLI.